compiler/resolution/src/org/jetbrains/kotlin/resolve/calls/tower/ImplicitScopeTower.kt

```diff
@@ -86,7 +86,7 @@ class CandidateWithBoundDispatchReceiver(
 )
 
 fun getResultApplicability(diagnostics: Collection<KotlinCallDiagnostic>) =
-    diagnostics.maxBy { it.candidateApplicability }?.candidateApplicability
+    diagnostics.maxByOrNull { it.candidateApplicability }?.candidateApplicability
```

Comment:
I didn't get why it's necessary

Reply:
Selected `Cleanup` option on commit (`maxBy` is deprecated)

Reply:
Reverted